src/lib/y2storage/free_disk_space.rb

```diff
@@ -50,6 +50,13 @@ def initialize(disk, region)
       @region = Y2Storage::Region.new(region)
     end
 
+    # Whether this free space belongs to an implicit partition table
+    #
+    # @return [Boolean]
+    def in_implicit_partition_table?
```

Comment:
I would not mention the implicit partition table as part of the name. That's the "why" (the only "why" we have so far but maybe not the only reason to reuse partitions in the future).

The "what" is the fact that this space is not really an unpartitioned disk chunk, but a partition that needs to be reused as-is.

So I would go with something as explicit as `#partition_to_reuse?`

Reply:
And maybe we should add some information at the top class definition. Explaining that one of these objects can represent an unpartitioned chunk (maybe within and extended partition) or the space of a partition to be reused.